dashboard/test/controllers/home_controller_test.rb

```diff
@@ -124,6 +124,22 @@ def setup_user_with_gallery
     assert_select 'a[href="/admin"]'
   end
 
+  test 'do not show levelbuilder links when not levelbuilder' do
+    sign_in create(:user)
+
+    get :index
+    assert_select 'a[href="/levels/new"]', 0
```

Comment:
The partial is always rendered, with the existence of the `levelbuilder` permission changing whether the partial is empty or populated. So I don't think this works.